build.sh

```diff
@@ -60,17 +57,25 @@ packages() {
 
 	cd librespot
 
-	# Get the git rev of librespot for .deb versioning
-	LIBRESPOT_VER="$(git describe --tags "$(git rev-list --tags --max-count=1)" 2>/dev/null || echo unknown)"
+	# We really should use the git rev of librespot for .deb versioning.
+	# The latest tag as of writing (0.4.2) is misleading because it doesn't reflect
+	# the fact we are using the dev branch (staged for 0.5.0).
+	#
+	# Hardcoding 0.5.0-dev in combination with the git hash suffix seems to be an
+	# acceptable solution as long as the raspotify version also is bumped every
+	# time we update against librespot.
+	#
+	#LIBRESPOT_VER="$(git describe --tags "$(git rev-list --tags --max-count=1)" 2>/dev/null || echo unknown)"
+	LIBRESPOT_VER=0.5.0-dev
```

Comment:
Prefix `LIBRESPOT_VER` with a `v` to provide similarly named packages as before.